test/source/tests/settings.ts

```diff
@@ -970,6 +970,56 @@ export const defineSettingsTests = (testVariant: TestVariant, testWithBrowser: T
       await settingsPage.close();
     }));
 
+    ava.default('settings - ensure gracious behavior & ui should remain functional when updating client configuration', testWithBrowser(undefined, async (t, browser) => {
+      const acct = 'settings@settings.flowcrypt.test';
+      mockBackendData.clientConfigurationByAcctEmail[acct] = keyManagerAutogenRules;
```

Comment:
This would interfere with `settings - client configuration gets updated on settings and content script reloads` test if run in parallel, as there is the same `acct` used. So we either need a different `acct` for this test or finish #4734 first.

Reply:
You are right. Just changed `acct`.